### PR TITLE
conda-pkg-install-docs

### PR DESCRIPTION
**Pull request by maddenp-noaa**
**Synopsis**

Improve _User Guide_ > _Installation_ section. Preview [here](https://uwtools--382.org.readthedocs.build/en/382/)

**Type**

- [x] Documentation

**Impact**

- [x] This is a non-breaking change (existing functionality continues to work as expected)

**Checklist**

- [x] I have added myself and any co-authors to the PR's _Assignees_ list.
- [x] I have reviewed the documentation and have made any updates necessitated by this change.

### CODE REVIEW

File: docs/sections/user_guide/installation.rst [hunk 1]
```diff
@@ -1,42 +1,71 @@
...
-Users of the original Miniconda (or the :anaconda:`Anaconda distribution<>`) may need to add the flags ``-c conda-forge --override-channels`` to ``conda build``, ``conda create``, and ``conda install`` commands to specify the use of conda-forge packages.
+Use an Existing conda Installation
+----------------------------------
 
-Using a Fresh Miniforge Installation
-------------------------------------
+Install Into an Existing Environment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To install ``uwtools`` into an exiting environment in an existing conda (e.g. :miniforge:`Miniforge<>`, :miniconda:`Miniconda<>`, :anaconda:`Anaconda<>`) installation:
```
Comment (christinaholtNOAA) [inline docs/sections/user_guide/installation.rst:14]: exiting -> existing

File: docs/sections/user_guide/installation.rst [hunk 2]
```diff
@@ -1,42 +1,71 @@
...
 
 .. include:: ../../shared/miniforge3_instructions.rst
 
-2. Install the ``conda-build`` and ``conda-verify`` packages into the base environment. If ``conda-build`` and ``conda-verify`` are already installed in your installation’s base environment, you may skip this step.
+2. Continue with the `Use an Existing conda Installation`_ instructions.
+
+Build the ``uwtools`` Package Locally
+-------------------------------------
+
+1. Install the necessary build packages into the conda installation's base environment (see the `Use a Fresh Miniforge Installation`_ instructions if an installation is unavailable or not writable):
```
Comment (christinaholtNOAA) [inline docs/sections/user_guide/installation.rst:48]: Does it make sense to use the `#` notation for numbered lists here since the numbers changed?

**Review by christinaholtNOAA — APPROVED** [review]
Thanks so much for the speedy response!

This looks great. Very minor comments below.

**Review by WeirAE — APPROVED** [review]
Minor visual question but otherwise LGTM

File: docs/sections/user_guide/installation.rst (hunk 1 above)
Comment (maddenp-noaa) [reply docs/sections/user_guide/installation.rst:14]: Fixed.

File: docs/sections/user_guide/installation.rst [hunk 3]
```diff
@@ -1,42 +1,71 @@
...
+The recommended installation mechanism uses the Python package and virtual-environment manager :conda:`conda<>`. Specifically, these instructions assume use of the :miniforge:`Miniforge<>` variant of :miniconda:`Miniconda<>`, built to use, by default, packages from the :conda-forge:`conda-forge<>` project. Users of the original :miniconda:`Miniconda<>` or the :anaconda:`Anaconda distribution<>` should add the flags ``-c conda-forge --override-channels`` to ``conda`` commands to specify the required package channels.
 
-Users of the original Miniconda (or the :anaconda:`Anaconda distribution<>`) may need to add the flags ``-c conda-forge --override-channels`` to ``conda build``, ``conda create``, and ``conda install`` commands to specify the use of conda-forge packages.
+Use an Existing conda Installation
+----------------------------------
 
-Using a Fresh Miniforge Installation
-------------------------------------
+Install Into an Existing Environment
```
[the remark this answers is not in the document]
Comment (maddenp-noaa) [reply docs/sections/user_guide/installation.rst:11]: These are section headings, so we're limited to using what Sphinx supports, and I don't think indentation or bullets are an option for section headings. Just to be sure we're on the same page, are you talking about this?
image

File: docs/sections/user_guide/installation.rst (hunk 2 above)
Comment (maddenp-noaa) [reply docs/sections/user_guide/installation.rst:48]: The auto-numbering `#.` syntax seems to be thrown off by various things, for example `code-block` entries between items. We already had this problem with the `include`d Miniforge installation snippet, which provides item `1.` -- Sphinx did not automatically number the next bullet at `2.`, but restarted from `1.` (same thing happens with interspersed code blocks). Maybe there's a workaround for this, but I haven't found it yet. I like `#.`, but it seems to work as expected only in simple/limited cases.

Comment (maddenp-noaa) [reply docs/sections/user_guide/installation.rst:48]: I'll add that I've had the same problem in the past with Markdown numbered lists, and I get it: It's not necessarily possible to know when a list is continuing vs when a new list is being started.

**Pull request merged**